.vscode/launch.json

```diff
@@ -1,11 +1,13 @@
 {
     "version": "0.2.0",
     "configurations": [
+        
+
         {
             "name": "(gdb) traffic_server",
             "type": "cppdbg",
             "request": "launch",
-            "program": "${workspaceFolder}/${env:ATS_VSCODE_BUILDDIR}/src/traffic_server/traffic_server",
+            "program": "/tmp/ats-dev/bin/traffic_server",
```

Comment:
Why are you hardcoding this? Could this not ever be installed in a different directory?

Reply:
Resolved